deployment/nginx/nginx.conf

```diff
@@ -12,6 +12,15 @@ http {
     default_type application/octet-stream;
     access_log /var/opt/rh/rh-nginx112/log/nginx/access.log combined;
 
+    map $http_origin $DO_CORS {
+        hostnames;
```

Comment:
It's been a while since I had my head in the nginx config space. What does `hostnames;` do here?

Reply:
```
hostnames
indicates that source values can be hostnames with a prefix or suffix mask
```
http://nginx.org/en/docs/http/ngx_http_map_module.html